pyproject.toml

```diff
@@ -16,14 +16,14 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-numpy = "^1.19.4"
-torch = "^1.7.1"
-matplotlib = "^3.3.3"
-pytorch-lightning = "^1.1.2"
-sklearn = "^0.0"
-torchvision = "^0.8.2"
-importlib_resources = "^5.0.0"
-tqdm = "^4.56.0"
+numpy = "^1.26.3"
+torch = "^2.1.2"
+matplotlib = "^3.8.2"
+pytorch-lightning = "^2.1.3"
+sickit_learn = "^1.4.0"
```

Comment:
Ah missed it, this is wrong. You should also run poetry lock after you update deps here

Reply:
Yep I know but for some reason poetry won't run on my computer.. I'll try to fix it.